.github/workflows/appimage.yml

```diff
@@ -26,12 +26,25 @@ jobs:
     timeout-minutes: 30
 
     steps:
-    - name: Check out code in viam-cartographer directory
+    - name: Check out main code
+      if: github.event_name == 'workflow_dispatch' || github.event_name == 'push'
```

Comment:
When will `github.event_name == 'workflow_dispatch'` evaluate to true?

Reply:
workflow_dispatch is the trigger when this workflow is called from another. E.g. when another workflow "uses" it.